packages/perennial-oracle/contracts/ChainlinkFeedOracle.sol

```diff
@@ -70,24 +70,42 @@ contract ChainlinkFeedOracle is IOracleProvider {
         // Revert if the round id is 0
         if (uint64(round.roundId) == 0) revert InvalidOracleRound();
 
-        // If there is more than 1 phase to update, revert
```

Comment:
update the `@dev` note for the function to reflect new logic

Reply:
can we add some documentation here to clarify that:
- `lastSyncedRoundId` -> latest previously seen round
- `nextPhase` -> phase where we're searching for round `n + 1`, is `nextStartingRoundId = n+1`?
- `round.phaseId()` -> current round

This took me a bit to figure out.